### PR TITLE
Fix the compile error #2930

### PR DESCRIPTION
**Pull request by dev4u**

### CODE REVIEW

File: core/build.gradle.kts [hunk 1]
```diff
@@ -51,9 +51,17 @@ cargo {
         "aead-cipher-2022",
     ))
     exec = { spec, toolchain ->
-        spec.environment("RUST_ANDROID_GRADLE_PYTHON_COMMAND", "python3")
-        spec.environment("RUST_ANDROID_GRADLE_LINKER_WRAPPER_PY", "$projectDir/$module/../linker-wrapper.py")
-        spec.environment("RUST_ANDROID_GRADLE_TARGET", "target/${toolchain.target}/$profile/lib$libname.so")
+        run {
+            val process: Process = Runtime.getRuntime().exec("which python3 >/dev/null 2>&1 && echo python3 installed")
```
Comment (madeye) [inline core/build.gradle.kts:55]: I don't think 'which' can work on Windows?

Comment (dev4u) [reply core/build.gradle.kts:55]: 噢，我改一下。

**Comment by dev4u** [comment]
@madeye 已经修改，请检视。

**Comment by madeye** [comment]
I didn't really get the idea of this fix. You are assuming python or python3 should be installed?

Maybe output an error message is more friendly.

**Comment by dev4u** [comment]
好的，我明白你的意思。
我回去再改一下。

**Comment by dev4u** [comment]
@madeye how about this latest commit?

**Comment by madeye** [comment]
merged, thanks.

**Pull request merged**